polynote-frontend/polynote/ui/component/plot_editor.ts

```diff
@@ -212,6 +212,8 @@ export class PlotEditor extends EventTarget {
         this.plotTypeSelector.addListener(() => this.onPlotTypeChange());
 
         this.el.addEventListener('dragstart', evt => {
+           // Firefox only allows dragging elements with a set DataTransfer
+           evt && evt.dataTransfer && evt.dataTransfer.setData('dummy', 'dummy');
```

Comment:
Is `dataTransfer` a standard API for drag-n-drop? Maybe we should be using that instead of jamming properties onto the drag element and other mutable state junk?

Regardless this seems like a harmless change so I'll go ahead and merge it.

Reply:
I think so. From the small number of examples that I saw, I think that the "correct way" to do this is to send the element id via data transfer (I'm no web dev though, so this might be completely wrong).

I did notice that the first field (the data type) can affect the drag and drop behavior (e.g. if it's `data/text`, it can be dragged onto the address bar). I have no idea what's the correct data type in this case.